diff --git a/minisat/core/Dimacs.h b/minisat/core/Dimacs.h
index d5db413..e753fb0 100644
--- a/minisat/core/Dimacs.h
+++ b/minisat/core/Dimacs.h
@@ -47,7 +47,7 @@ static void readClause(B& in, Solver& S, vec<Lit>& lits) {
 template<class B, class Solver>
 static void parse_DIMACS_main(B& in, Solver& S, bool strictp = false) {
     vec<Lit> lits;
-    int vars    = 0;
+    // int vars    = 0;
     int clauses = 0;
     int cnt     = 0;
     for (;;){
@@ -55,7 +55,7 @@ static void parse_DIMACS_main(B& in, Solver& S, bool strictp = false) {
         if (*in == EOF) break;
         else if (*in == 'p'){
             if (eagerMatch(in, "p cnf")){
-                vars    = parseInt(in);
+                // vars    = parseInt(in);
                 clauses = parseInt(in);
                 // SATRACE'06 hack
                 // if (clauses > 4000000)
